influxdb_client/client/flux_csv_parser.py

```diff
@@ -1,6 +1,6 @@
 """Parsing response from InfluxDB to FluxStructures or DataFrame."""
 
-
+import asyncio
```

Comment:
`sync` client doesn't include `asyncio` library. You have to check the type by name... maybe something like:

https://github.com/influxdata/influxdb-client-python/blob/3d70dbd66e434d5ac58e8d018e566d3089223db3/influxdb_client/client/_base.py#L485